ote_cli/ote_cli/tools/train.py

```diff
@@ -138,12 +145,20 @@ def main():
     )
 
     if args.load_weights:
+        model_adapters = {
+            "weights.pth": ModelAdapter(read_binary(args.load_weights)),
+        }
+        if osp.exists(osp.join(osp.dirname(args.load_weights), "label_schema.json")):
+            model_adapters.update({
+                    "label_schema.json": ModelAdapter(
+                        label_schema_to_bytes(read_label_schema(args.load_weights))
+                    )
+                }
+            )
         environment.model = ModelEntity(
             train_dataset=dataset,
             configuration=environment.get_model_configuration(),
-            model_adapters={
-                "weights.pth": ModelAdapter(read_binary(args.load_weights))
-            },
+            model_adapters=model_adapters,
```

Comment:
It is natural to load label schema on which the model weights had been trained, if any. Class-Incremental learning exploits the gap between the global schema and the model schema.